processing/data_collection/gazette/settings.py

```diff
@@ -8,4 +8,4 @@
     "gazette.pipelines.ExtractTextPipeline": 200,
 }
 FILES_STORE = "/mnt/data/"
-QUERIDODIARIO_EXTRACT_TEXT_FROM_FILE = True
\ No newline at end of file
+QUERIDODIARIO_EXTRACT_TEXT_FROM_FILE = True
```

Comment:
this same change is in another of your PRs, it will cause conflicts once that one is merged

Reply:
Yes, but If I remove it, the automated build will fail.

Reply:
It will not fail once one is merged, so after you can rebase it, maybe just wait a bit to get one merged and then fix the others.

Reply:
That was the plan. Have the first one merged, then rebase the others. Can we start with this one?